src/sasctl/pzmm/write_json_files.py

```diff
@@ -639,19 +616,19 @@ def generateROCLiftStat(
         targetValue: int or float
             Value of target variable that indicates an event.
         swatConn: SWAT connection to CAS
-            Connection object to CAS service in SAS Model Manager or SAS® Open Model Manager through SWAT authentication.
+            Connection object to CAS service in SAS Model Manager or SAS® Open Model Manager through SWAT
+            authentication.
         validateData : pandas dataframe, numpy array, or list, optional
-            Dataframe, array, or list of the validation data set, including both
-            the actual values and the calculated probabilities. The default value is None.
+            Dataframe, array, or list of the validation data set, including both the actual values and the calculated
+            probabilities. The default value is None.
         trainData : pandas dataframe, numpy array, or list, optional
-            Dataframe, array, or list of the train data set, including both
-            the actual values and the calculated probabilities. The default value is None.
+            Dataframe, array, or list of the train data set, including both the actual values and the calculated
+            probabilities. The default value is None.
         testData : pandas dataframe, numpy array, or list, optional
-            Dataframe, array, or list of the test data set, including both
-            the actual values and the calculated probabilities. The default value is None.
+            Dataframe, array, or list of the test data set, including both the actual values and the calculated
+            probabilities. The default value is None.
         jPath : string, optional
-            Location for the output JSON file. The default value is the current
-            working directory.
+            Location for the output JSON file. The default value is the current working directory.
 
         Yields
```

Comment:
Same issue as with writeVarJSON() docstring, the Yields section is inappropriate here.

Reply:
Resolved.